gnome-keysign.py

```diff
@@ -130,7 +137,7 @@ def __init__(self, keyid, keydata):
         self.data = keydata
 
         label = Gtk.Label()
-        label.set_markup(keydata)
+        label.set_markup(keydata.replace('<', '').replace('>', ''))
```

Comment:
On Mon, Jul 18, 2016 at 04:18:46AM -0700, Andrei Macavei wrote:
> But if you replace the markup tags ('<', '>') then the text will not use them anymore. Here this will show `small` in the text label.
Sure. This is quick and dirty to make it work.
You need to fix that up ;-)
Feel free to entirely ignore these changes and re-do them.
The main point shall be the way to get hold of the keys.